git-bulk: add page (#8946)
tldr-pages/tldr: pages/common/git-bulk.md

# git bulk

> Execute operations on multiple git repositories.
> Part of `git-extras`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-bulk>.

- Register a new workspace into git-bulk:

`git bulk --addworkspace {{workspace_name}} {{absolute/path/to/root/directory}}`

> - Register a new workspace and a git repository attached to it:

Reviewer: Perhaps this is a better explanation
```suggestion
- Clone a repository inside a specific directory then register the repository as a workspace:
```

`git bulk --addworkspace {{workspace-name}} {{absolute/path/to/root/directory}} --from {{git_repo_url}}`

> - Register a new workspace and attach some git repositories from a file:

Reviewer: ```suggestion
- Clone repositories from a newline-separated list of remote locations then register them as workspaces:
```

`git bulk --addworkspace {{workspace-name}} {{absolute/path/to/root/directory}} --from {{absolute/path/to/file}}`

- List all registered workspaces:

`git bulk --listall`

> - Run a git command on the repositories of the current workspace:

Reviewer: ```suggestion
- Run a Git command on the repositories of the current workspace:
```

> `git bulk {{git_command}}`

Reviewer: ```suggestion
`git bulk {{command}} {{command_arguments}}`
```

> `git bulk {{commmand}} {{commmand_arguments}}`

Reviewer: ```suggestion
`git bulk {{command}} {{command_arguments}}`
```
